src/Rings/mpoly-graded.jl

```diff
@@ -1667,7 +1666,7 @@ function _divide_by(a::Vector{Vector{Int}}, pivot::Vector{Int})
     # the next line computers   m = [k < 0 ? 0 : k for k in e]
     # but without allocations
     for i in 1:length(e)
-      m[i] = e[i] >= 0 ? e[i] : 0
+      m[i] = max(e[i] - pivot[i], 0)
```

Comment:
Just out of curiosity: Why would this be faster?

Reply:
I did not check if it really is faster (or slower! for that matter); but without the `max`, this would be
```suggestion
      m[i] = e[i] - pivot[i] >= 0 ? e[i] - pivot[i] : 0
```
so it potentially computes `e[i] - pivot[i]` twice. But perhaps Julia's optimizer will avoid this via common code elimination; but I also found the version with `max` more readable.